sci-electronics/fritzing/fritzing-0.9.2b-r1.ebuild

```diff
@@ -0,0 +1,73 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+PARTS_P="${PN}-parts-${PV}"
+
+inherit qmake-utils
+
+DESCRIPTION="Electronic Design Automation"
+HOMEPAGE="http://fritzing.org/"
+SRC_URI="https://github.com/fritzing/fritzing-app/archive/${PV}.tar.gz -> ${P}.tar.gz
+	https://github.com/fritzing/fritzing-parts/archive/${PV}.tar.gz -> ${PARTS_P}.tar.gz"
+
+LICENSE="CC-BY-SA-3.0 GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+RDEPEND="dev-qt/qtconcurrent:5
+	dev-qt/qtcore:5
+	dev-qt/qtgui:5
+	dev-qt/qtnetwork:5
+	dev-qt/qtprintsupport:5
+	dev-qt/qtserialport:5
+	dev-qt/qtsql:5[sqlite]
+	dev-qt/qtsvg:5
+	dev-qt/qtwidgets:5
+	dev-qt/qtxml:5
+	dev-libs/quazip[qt5]"
+DEPEND="${RDEPEND}
+	>=dev-libs/boost-1.40"
+
+S="${WORKDIR}/${PN}-app-${PV}"
+
+src_prepare() {
+	local translations=
+
+	# Get a rid of the bundled libs
+	# Bug 412555 and
+	# https://code.google.com/p/fritzing/issues/detail?id=1898
+	rm -rf src/lib/quazip/ pri/quazip.pri src/lib/boost* || die
+
+	# Fritzing doesn't need zlib
+	sed -i -e 's:LIBS += -lz::' -e 's:-lminizip::' phoenix.pro || die
+
+	# Somewhat evil but IMHO the best solution
+	for lang in $LINGUAS; do
+		lang=${lang/linguas_}
+		[ -f "translations/${PN}_${lang}.qm" ] && translations+=" translations/${PN}_${lang}.qm"
+	done
+	if [ -n "${translations}" ]; then
+		sed -i -e "s:\(translations.extra =\) .*:\1	cp -p ${translations} \$(INSTALL_ROOT)\$\$PKGDATADIR/translations\r:" phoenix.pro || die
+	else
+		sed -i -e "s:translations.extra = .*:\r:" phoenix.pro || die
+	fi
+
+	# Fix missing Intel bin icon
+	# https://github.com/fritzing/fritzing-parts/commit/716560e9
+	sed -i -e 's:Intel.png:intel.png:' bins/more/intel.fzb || die
+}
+
+src_configure() {
+	eqmake5 DEFINES=QUAZIP_INSTALLED phoenix.pro
+}
+
+src_install() {
+	INSTALL_ROOT="${D}" emake install
```

Comment:
have you checked whether `INSTALL_ROOT` is really needed instead of `DESTDIR`? If `DESTDIR` also works, switch this to `default`